docs/api/browser-window.md

```diff
@@ -379,6 +379,8 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
     * `disableHtmlFullscreenWindowResize` Boolean (optional) - Whether to
       prevent the window from resizing when entering HTML Fullscreen. Default
       is `false`.
+    * `accessibleTitle` String (optional) - a part of Window title visible for
```

Comment:
`String (optional) - An alternative title string provided only to accessibility tools such as screen readers.  This string is not directly visible to users.`